bindings/python/iota_sdk/wallet/account.py

```diff
@@ -1,34 +1,36 @@
-from iota_wallet.common import send_message_routine
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
 
+from iota_sdk.wallet.common import _call_method_routine
 
 class Account:
     def __init__(self, account_id, handle):
         self.account_id = account_id
         self.handle = handle
 
-    @send_message_routine
+    @_call_method_routine
```

Comment:
Can you remind me why it's needed then pls?

Reply:
Because `_call_method()` of the wallet creates a different message than `_call_account_method()`, but both send it over the same function